src/components/task/index.cjsx

```diff
@@ -54,24 +42,24 @@ module.exports = React.createClass
   render: ->
     {id} = @props
     model = TaskStore.get(id)
-    steps = TaskStore.getSteps(id)
-    stepConfig = steps[@state.currentStep]
+    stepConfig = @state.steps[@state.currentStep]
+
     {courseId} = @context.router.getCurrentParams()
 
-    allStepsCompleted = true
-    for step in steps
-      unless step.is_completed
-        allStepsCompleted = false
+    allStepsCompleted = TaskStore.isTaskCompleted(id)
 
-    if steps.length > 1
+    classes = classNames({
```

Comment:
I'm fine with this use for setting class names but to keep the code a little more consistent, could you either make this change for all places that require logic when setting the `className` or use methods already in the codebase?